quipucords/scanner/network/processing/test_installed_products.py

```diff
@@ -0,0 +1,68 @@
+"""Unit tests for installed_products fact initial processing."""
+
+from scanner.network.processing import installed_products
+from scanner.network.processing.util_for_test import ansible_result
+
+
+def test_success_found_id_and_name():
+    """ID and Name match expected format."""
+    stdout_output = (
+        "Product:\r\n\tID: 69\r\n\tName: Red Hat Enterprise Linux Server\r\n\t"
+        "Version: 7.0\r\n\t"
+    )
+    expected = [{"id": "69", "name": "Red Hat Enterprise Linux Server"}]
+    assert (
+        installed_products.ProcessInstalledProducts.process(
+            ansible_result(stdout_output)
+        )
+        == expected
+    )
+
+
+def test_success_found_id():
+    """Only ID matches expected format."""
+    stdout_output = (
+        "Product:\r\n\tName: Red Hat Enterprise Linux Server\r\n\tID: 69\r\n\t"
+        "Version: 7.0\r\n\t"
```

Comment:
Since we are using grep with `-C`, I think we should take advantage of its delimiter to separate things.